demisto_sdk/commands/validate/file_validator.py

```diff
@@ -523,13 +527,121 @@ def validate_committed_files(self):
 
     def validate_pack_unique_files(self, packs):
         for pack in packs:
+            print(f'Validating {pack} unique pack files')
             pack_unique_files_validator = PackUniqueFilesValidator(pack)
             pack_errors = pack_unique_files_validator.validate_pack_unique_files()
             if pack_errors:
                 print_error(pack_errors)
                 self._is_valid = False
 
+    def file_all_validations(self, file_path, file_type: str = None):
+        file_extention = os.path.splitext(file_path)[-1]
+        if re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE) and not file_type:
+            return
+        if file_extention not in ['.yml', 'json', '.md']:
+            return
+        # Ignoring changelog and description files
+        if 'changelog' in file_path.lower() or 'description' in file_path.lower():
+            return
+        print(f'Validating {file_path}')
+        if re.match(TEST_PLAYBOOK_REGEX, file_path, re.IGNORECASE):
```

Comment:
should include minimal validator checks (`id == name`, `version == -1`)

Reply:
this can be improved in another PR

Reply:
I'll have a commit for that